base/error.jl

```diff
@@ -52,28 +52,31 @@ end
 
 
 """
-    retry(f, [condition]; n=3; max_delay=10) -> Function
```

Comment:
_Note on implementation detail: the `max_delay` cap in `retry` only takes effect after the first delay. So setting `max_delay=0` results in a ~50ms delay after the first attempt, and 0 delay on after subsequent attempts._